src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.impl.response.ShoppingCartMapper;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/shopping-carts")
+public class ShoppingCartController {
+    private final ShoppingCartService shoppingCartService;
+    private final MovieSessionService movieSessionService;
+    private final UserService userService;
+    private final ShoppingCartMapper shoppingCartMapper;
+
+    public ShoppingCartController(ShoppingCartService shoppingCartService,
+                                  MovieSessionService movieSessionService,
+                                  UserService userService,
+                                  ShoppingCartMapper shoppingCartMapper) {
+        this.shoppingCartService = shoppingCartService;
+        this.movieSessionService = movieSessionService;
+        this.userService = userService;
+        this.shoppingCartMapper = shoppingCartMapper;
+    }
+
+    @PutMapping("/movie-sessions")
+    public String addSessionToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
+        shoppingCartService.addSession(movieSessionService.get(movieSessionId),
+                userService.get(userId));
+        return "Ticket was added to your shopping card";
```

Comment:
it'd be better to give back a status code in a response https://restfulapi.net/http-methods/#:~:text=If%20an%20existing%20resource%20is%20modified%2C%20either%20the%20200%20(OK)%20or%20204%20(No%20Content)%20response%20codes%20SHOULD%20be%20sent%20to%20indicate%20successful%20completion%20of%20the%20request.